shell/platform/android/io/flutter/embedding/engine/plugins/util/GeneratedPluginRegister.java

```diff
@@ -14,16 +14,57 @@
   /**
    * Registers all plugins that an app lists in its pubspec.yaml.
    *
-   * <p>The Flutter tool generates a class called GeneratedPluginRegistrant, which includes the code
-   * necessary to register every plugin in the pubspec.yaml with a given {@code FlutterEngine}. The
-   * GeneratedPluginRegistrant must be generated per app, because each app uses different sets of
-   * plugins. Therefore, the Android embedding cannot place a compile-time dependency on this
-   * generated class. This method uses reflection to attempt to locate the generated file and then
-   * use it at runtime.
+   * <p>To let each plugin set itself up to listen to usage calls from Dart via platform channel,
+   * each plugin is given a chance to initialize itself and setup platform channel listeners in
+   * {@link
+   * io.flutter.embedding.engine.plugins.FlutterPlugin#onAttachedToEngine(io.flutter.embedding.engine.plugins.FlutterPlugin.FlutterPluginBinding)}.
+   *
+   * <p>The list of plugins needing setup is not known to the Flutter engine, but generated at build
```

Comment:
prefer present tense: The list of plugins that **need to be setup** is not known to the Flutter engine.   Flutter **generates** it at build time...